src/PKSim.Core/Snapshots/Mappers/SimulationMapper.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Repositories;
+using OSPSuite.Utility.Extensions;
+using PKSim.Core.Model;
+using PKSim.Core.Repositories;
+using SnapshotSimulation = PKSim.Core.Snapshots.Simulation;
+using ModelSimulation = PKSim.Core.Model.Simulation;
+
+namespace PKSim.Core.Snapshots.Mappers
+{
+   public class SimulationMapper : ObjectBaseSnapshotMapperBase<ModelSimulation, SnapshotSimulation, PKSimProject>
```

Comment:
This is a best of a mapper. Some stuff could be put in specific mapper such as event, but also event mapping is only used in the context of a simulation and is a one liner. So in the spirit of keeping it simple, no mapper for now